db/note.php

```diff
@@ -67,6 +94,28 @@ public static function fromFile(File $file, Folder $notesFolder, $tags=[]){
         $note->resetUpdatedFields();
         return $note;
     }
+    /**
+     * @param File $file
+     * @return static
+     */
+    public static function fromException($message,File $file,Folder $notesFolder,$tags=[]){
+        $note = new static();
+        $note->setId($file->getId());
+        $note->setErrorMessage($message);
+        $note->setError(true);
+        $note->setContent($message);
+        $note->setModified(null);
```

Comment:
The Android app (accidentally) ignores notes with `modified=null`. Hence, note errors will not show up in the Android app. This may be intended (otherwise, the Android app has to deal with such special notes). However, the erroneous notes can only be detected in the web app (which is not a degradation to the current situation!). Is this what we want? (I think it's okay).
/cc @stefan-niedermann

Reply:
I'm not near an Android device, but I'll check this out when I am. The error messages have more or less the same structure as the ordinary notes, so I think it might be possible for the Android app to recognise them.

Reply:
Yes, if `modified!=null`, then the error message looks like a real note. Hence, the app allows the user to open and edit the error message. This is not useful, since saving will obviously fail. I see two posibilities:

1. Stay with `modified=null`, no changes in the Android app. Faulty notes are not shown to the user.
2. Set `modified=filemtime()` and change the Android app, so that notes with error are shown read-only.

/cc @jancborchardt